docs/root/intro/version_history.rst

```diff
@@ -108,6 +108,8 @@ Version history
   is supplied with the client request, its value will override any sampling decision made by the Envoy proxy.
 * websocket: support configuring
   :ref:`idle_timeout and max_connect_attempts <envoy_api_field_route.RouteAction.websocket_config>`.
+* tls: added support for allowing expired certificates with
```

Comment:
Nit: release notes is configured alphabetically, this should probably be added just before `tls: removed...`.